sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/complexTypeExtractors.scala

```diff
@@ -111,6 +111,11 @@ case class GetStructField(child: Expression, ordinal: Int, name: Option[String]
   override def dataType: DataType = childSchema(ordinal).dataType
   override def nullable: Boolean = child.nullable || childSchema(ordinal).nullable
 
+  override def verboseString: String = {
```

Comment:
We rarely call directly `Expression.verboseString`. It is mostly called by `treeString` to show individual nodes in this tree.